sonaranalyzer-dotnet/src/SonarAnalyzer.CFG/ControlFlowGraph/CSharpControlFlowGraph.cs

```diff
@@ -50,9 +50,7 @@ public static bool TryGet(CSharpSyntaxNode node, SemanticModel semanticModel, ou
                                         exc is ArgumentException ||
                                         exc is NotSupportedException)
             {
-                // historically, these have been considered as expected
-                // but we should be aware of what syntax we do not yet support (TODO)
-                // https://github.com/SonarSource/sonar-dotnet/issues/2541
+                // These are expected
```

Comment:
This comment also seems to be something that was missed when backporting changes to the security CFG. The removed comment with the ToDo and github issue link is actually more recent, so it would be better to keep that one